compressor_test.go

```diff
@@ -27,12 +28,34 @@ func TestSnappyCompressor(t *testing.T) {
 		t.Fatalf("failed to encode '%v' with error '%v'", str, err)
 	}
 
-	//Test Decoding
+	//Test Decoding with Snappy
 	if expected, err := snappy.Decode(nil, val); err != nil {
 		t.Fatalf("failed to decode '%v' with error %v", val, err)
 	} else if res, err := c.Decode(val); err != nil {
 		t.Fatalf("failed to decode '%v' with error %v", val, err)
 	} else if bytes.Compare(expected, res) != 0 {
 		t.Fatal("failed to match the expected decoded value with the result decoded value.")
 	}
+
+	//Test Encoding with S2
+	expected = s2.EncodeSnappy(nil, []byte(str))
+	if res, err := c.Encode([]byte(str)); err != nil {
+		t.Fatalf("failed to encode '%v' with error %v", str, err)
+	} else if bytes.Compare(expected, res) != 0 {
```

Comment:
Are we verifying that S2 can indeed produce a Snappy compatible output, or a byte to byte compatible output? It seems the latter, which isn't clear is what S2 can do actually. I think what you need to test is can you decode it.

(But I also think it's not needed. S2 probably has such basic tests).

Reply:
this test doesn't make sens I removed it